telemetrylib/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(CMAKE_AUTOMOC ON)
+
+cmake_minimum_required(VERSION 3.20)
+project(telemetrylib)
+
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Changed, this cmake file compile the telemetrylib itself, but since QT is giving me a hard time importing the library, the code in telemetrylib is currently being compiled as part of the source code of driver io using QMake.